certified-connectors/Exasol/README.md

```diff
@@ -0,0 +1,34 @@
+
+## Exasol - The analytics database connector
+
+Our superfast data analytics platform gives you the power, flexibility and scalability to meet the demands of your long-term data strategy. 
+
```

Comment:
@pj-spoelders Please refer [the template for readme.md file](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md) and add required secitons.

Reply:
Thanks for bringing this template to our attention. I altered the document and added the missing sections.